price-estimator/docker/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+image_name=$1
+
+echo "Docker login"
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin;
+
+echo "Building price estimator docker";
+docker build --tag gnosispm/dex-price-estimator-rust:$image_name -f price-estimator/docker/Dockerfile .
+
+echo "Pushing price estimator docker"
+docker push gnosispm/dex-price-estimator-rust:$image_name;
```

Comment:
nit: I would just call it `dex-price-estimator`, I feel like it being Rust is an implementation detail that can be encoded in the tag name.

Reply:
The reasoning for this name is that the nodejs price estimator image is called `dex-price-estimator`. I did not want to replace that name so I appended `rust`.

Reply:
I would argue that we are replacing the nodejs price estimator with the Rust one, so we should re-appropriate the image name. I think having different tags is enough to differentiate the different versions.

We could even create a `:legacy` tag just for the latest nodejs version so its archived and easily retrievable.

Reply:
Just to be safe I don't want to every accidentally override one image with the other (like we bump the old price estimator for some reason and it overrrides the master tag). So I'd rather have separate names.

Reply:
What about `image_tag="$1-rust"`? IMO its better to have more tags and a single image than a bunch of images.

For example, the NodeJS image does this to select between [different operating systems](https://hub.docker.com/_/node/). Especially because these are intended to be the same service, it makes more sense for them to have the same name.

Reply:
But then it's an exception in our images that this one doesn't use "master" or the git tag as the image tag. Seems like more disadvantages just to avoid an imperfect name.

Reply:
I disagree, but I also don't think this super important. I'm fine with `dex-price-estimator-rust` but suggest revisiting this if/when we deprecate and archive the gnosis/dex-price-estimator so this wouldn't be a concern any more.